content/pages/project.rst

```diff
@@ -27,10 +27,15 @@ modmesh
         </span>
     </p>
 
-- **Repository Link:** `Github <https://github.com/solvcon/modmesh>`__
+- **Repository Link:** `GitHub <https://github.com/solvcon/modmesh>`__
 - **Leader:** `Yung-Yu Chen <https://twitter.com/yungyuc>`__
 
-Toolkit for solving partial differential equations and analysis of the results
+modmesh seamlessly mixes C++ and Python through pybind11, allowing you to leverage the strengths of 
+both programming languages for efficient PDE solving. We use Qt to visualize the computation results 
+to make you understand your PDE solution. We recently tried to add Gmsh to load the custom mesh to solve 
+the PDE problem. 
+
+Modmesh is designed to run on Windows, Linux, and macOS. Everyone can use or contribute to modmesh.
```

Comment:
@kuomc, Could you please use the following paragraph to instead of origin paragraph, thanks for the help!

modmesh seamlessly mixes C++ and Python by pybind11, allowing you to leverage the strengths of bot programming languages for efficient PDE solving. We use Qt and Python to visualize the computation results to let you understand your PDE solution. modmesh also support mesh visualization, currently support Gmsh mesh file format.
We recently tried to improve modmesh UI/UX.
modmesh is designed to run on Windows, Linux and MacOS. Everyone can use or contribute to modmesh.

modmesh seamlessly mixes C++ and Python through pybind11, allowing you to leverage the strengths of both programming languages for efficient PDE solving. We use Qt and Python to visualize the computation results to make you better understanding of your PDE solution. modmesh also supports mesh visualization, currently in the Gmsh mesh file format. We have recently made efforts to improve the modmesh UI/UX.

The design allows it to run on Windows, Linux, and MacOS. Everyone can use or contribute to modmesh.